css/styles.css

```diff
@@ -96,6 +97,226 @@ ul {
   justify-content: start;
 }
 
+/* My Works Section */
+
+.my-works {
```

Comment:
- [ ] The my-works section should have margins as defined in the Figma design. I suggest you add margins to the styling.